<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>flightDetails</class>
 <widget class="QDialog" name="flightDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flight Details</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#flightDetails{

border-image:url(:/imagesMain/flightImage.jpg);

}

QLabel{

color:black;

}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#stackeWidget{

background:rgba(0,0,0,0);

}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>261</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Pastelle DEMO</family>
       <pointsize>35</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Booking Confirmed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>400</y>
       <width>431</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rage Italic</family>
       <pointsize>25</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>Thank you for choosing us. 
Happy Travels!</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>521</width>
       <height>291</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget{

border-image:url(:/imagesMain/plane.png);

}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>520</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>510</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm Booking</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>450</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>OCR A Extended</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Payment mode</string>
     </property>
    </widget>
    <widget class="QLabel" name="carrierName">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poor Richard</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Airline</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Nirmala UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>From:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fromLabel">
          <property name="font">
           <font>
            <family>Nirmala UI</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="dateTime_2">
          <property name="font">
           <font>
            <family>Nirmala UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ticket Fare:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="priceLabel">
          <property name="font">
           <font>
            <family>Nirmala UI</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Nirmala UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Date:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="date">
          <property name="font">
           <font>
            <family>Nirmala UI</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>250</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_12{

color:red;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>360</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_13{

color:red;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>400</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_14{

color:red;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>360</y>
       <width>421</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>E-mail ID:   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="email_ID"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>421</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>First Name:  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="firstName"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>210</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_11{

color:red;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Nirmala UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>To:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="toLabel">
        <property name="font">
         <font>
          <family>Nirmala UI</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>400</y>
       <width>421</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Phone number:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="phone"/>
      </item>
     </layout>
    </widget>
    <widget class="QComboBox" name="payment">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>450</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Visa</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MasterCard</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Net Banking</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>e-Wallet</string>
      </property>
     </item>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>310</y>
       <width>251</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Date of Birth </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>250</y>
       <width>421</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Last Name :  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lastName"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
